wire-ios/Tests/Mocks/MockConversation+factory.swift

```diff
@@ -63,4 +68,14 @@ extension MockConversation {
 
         return mockConversation
     }
+
+    @objc (willAccessValueForKey:)
+    func willAccessValue(forKey: String) {
+
+    }
+
+    @objc (didAccessValueForKey:)
+    func didAccessValue(forKey: String) {
+
+    }
```

Comment:
is it wanted? can we add a comment why the implementation is empty?

Reply:
https://github.com/wireapp/wire-ios/pull/1029

Reply:
there are failed tests without it